ResearchKit/Common/ORKImageCaptureStepViewController.m

```diff
@@ -109,26 +121,26 @@ - (void)retakePressed:(void (^ __nullable)())handler {
     });
 }
 
-- (void)capturePressed:(void (^ __nullable)(BOOL))handler {
+- (void)capturePressed:(void (^)(BOOL))handler {
     // Capture the image via the output
-    dispatch_async(self.sessionQueue, ^{
-    	[[self stillImageOutput] captureStillImageAsynchronouslyFromConnection:[[self stillImageOutput] connectionWithMediaType:AVMediaTypeVideo] completionHandler:^(CMSampleBufferRef imageDataSampleBuffer, NSError *error) {
-            [self queue_captureImageFromData:imageDataSampleBuffer withHandler:handler];
+    dispatch_async(_sessionQueue, ^{
+    	[_stillImageOutput captureStillImageAsynchronouslyFromConnection:[_stillImageOutput connectionWithMediaType:AVMediaTypeVideo] completionHandler:^(CMSampleBufferRef imageDataSampleBuffer, NSError *error) {
+            [self queueCaptureImageFromData:imageDataSampleBuffer handler:handler];
     	}];
     });
 }
 
 - (void)videoOrientationDidChange:(AVCaptureVideoOrientation)videoOrientation {
     // Keep the output orientation in sync with the input orientation
-    ((AVCaptureConnection *)self.stillImageOutput.connections[0]).videoOrientation = videoOrientation;
+    ((AVCaptureConnection *)_stillImageOutput.connections[0]).videoOrientation = videoOrientation;
 }
 
-- (void)queue_captureImageFromData:(CMSampleBufferRef)imageDataSampleBuffer withHandler:(void (^ __nullable)(BOOL))handler {
+- (void)queueCaptureImageFromData:(CMSampleBufferRef)imageDataSampleBuffer handler:(void (^)(BOOL))handler {
```

Comment:
Are you sure? Isn't this very much against the [Coding Guidelines for Cocoa](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html#//apple_ref/doc/uid/20001282-BCIGIJJF)?